### DISCUSSION TITLE
[Bug]: The provided host name is not valid for this server (Drupal site)

### DISCUSSION THREAD

**Discussion by shummel-brunner** [beyondcode/herd-community#91] [Q&A]
### Operating system version

macOS Ventura

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.1.1 (Bulld 12)

### PHP Version

PHP 8.0

### Bug description

When I try to open one of my Herd sites, I get the error "The provided host name is not valid for this server." All of my other sites work. This is a Drupal 9 site. Is that an issue for Herd? I expected it to load the site.

### Steps to reproduce

Add a Drupal site to Herd and try to open it.

### Relevant log output

```shell
Error message: "The provided host name is not valid for this server."
```

**Comment by mpociot** [comment]
I don't have enough Drupal knowledge to help you with this, so hopefully someone in the discussion can chime in.

**Reply by shummel-brunner** [reply]
What conditions causes that error message to display?

**Reply by shummel-brunner** [reply]
@mpociot Under what conditions would you hit that error messages? Where in the code does it throw that error? This may not be a Drupal-specific issues. Could you please track down that error message?
